src/FilteredActionList/FilteredActionList.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import {GroupedListProps, ListPropsBase} from '../ActionList/List'
+import TextInput from '../TextInput'
+import Box from '../Box'
+import SelectMenu from '../SelectMenu'
+import {ActionList} from '../ActionList'
+
+export interface FilteredActionListProps extends Partial<Omit<GroupedListProps, keyof ListPropsBase>>, ListPropsBase {
+  loading?: boolean
+  placeholderText: string
+  filterValue: string
+  onChange: (e: React.ChangeEvent<HTMLInputElement>) => unknown
+}
+
+export function FilteredActionList({
+  loading = false,
+  placeholderText,
+  filterValue,
+  onChange,
+  items,
+  ...listProps
+}: FilteredActionListProps): JSX.Element {
+  return (
+    <>
+      <TextInput
+        block
+        width="auto"
+        color="text.primary"
+        defaultValue={filterValue}
```

Comment:
I know we’d discussed using `TextInput` as a controlled component, which would require `value`, not `defaultValue`—where did we land on that?

Reply:
I'm going to remove the `filterValue` prop for now and have it be a 1-way piece of info that gets sent to the parent, but cannot be passed in.